package.json

```diff
@@ -54,5 +55,6 @@
     "prettier": "^3.2.5",
     "sass": "^1.69.5",
     "tailwindcss": "^3.3.5"
-  }
+  },
+  "packageManager": "yarn@1.22.22+sha512.a6b2f7906b721bba3d67d4aff083df04dad64c399707841b7acf00f6b133b7ac24255f2652fa22ae3534329dc6180534e98d17432037ff6fd140556e2bb3137e"
 }
```

Comment:
?

Reply:
The day that `yarn` will stop adding these to the `package.json` will mark a greater day in history than d-day.

Reply:
or move to yarn

Reply:
outside scope of this PR, I am not planning on making another 283-file commit

Reply:
agree